src/views/Layout.jsx

```diff
@@ -13,37 +12,33 @@ export function Layout({ lists, listPath }) {
 	};
 
 	return (
-		<div className="h-dvh flex flex-col text-poppins">
+		<div className="min-h-dvh w-screen xl:w-9/12 flex flex-col m-auto text-poppins relative">
```

Comment:
I think the `xl:w-9/12` is preventing the navbar to fill all the screen width in large screens:
<img width="1440" alt="Screenshot 2024-03-31 at 01 48 54" src="https://github.com/the-collab-lab/tcl-71-smart-shopping-list/assets/91918142/a833e9fa-178c-4770-9edc-b2fe1ad2a796">
Just pointing it out in case we want to make any changes now.

Reply:
It was actually what I intented, but now that I see in this morning, I will change that 😅

Reply:
@vivitt I updated the navbar. To keep the layout consitant, I put the nav bar as fixed and added a padding top for the main container.